api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -919,7 +961,7 @@ private Timestamp expired() {
   }
 
   private void advanceClockDays(long days) {
-    PROVIDED_CLOCK.setInstant(START_INSTANT.plus(days, ChronoUnit.DAYS));
+    PROVIDED_CLOCK.setInstant(Instant.now(PROVIDED_CLOCK).plus(days, ChronoUnit.DAYS));
```

Comment:
also a good catch!  I think we can remove the round-trip `Instant` conversion as well.  How about:
```
    PROVIDED_CLOCK.increment(Duration.ofDays(days).toMillis());
```